lib/Runtime/Types/EdgeJavascriptTypeId.h

```diff
@@ -19,115 +19,117 @@ enum TypeId
     TypeIds_Int64Number = 5,
     TypeIds_UInt64Number = 6,
     TypeIds_LastNumberType = TypeIds_UInt64Number,
+    TypeIds_BigInt = 7,
```

Comment:
is BigInt added here specifically because its a non-numeric primitive type?

Reply:
it is actually a numeric primitive type

Reply:
Should it go above then and get captured by `TypeIds_LastNumberType`?

Reply:
No, as the comment above says, typeof() == "number" is true for number types. This isn't a number, it's another kind of primitive. It's in the right place.

---
In reply to: [218221309](https://github.com/Microsoft/ChakraCore/pull/5706#discussion_r218221309) [](ancestors = 218221309)